components/atoms/ReportProblem.js

```diff
@@ -9,59 +15,169 @@ export default function ReportProblem() {
+              uncontrolled={true}
+              multiText={true}
+              textLabelBold={true}
+              wrap="hard"
+              maxLength={750}
+            />
+            <OptionalTextField
+              checkBoxId="adaptiveTechnology"
+              textFieldId="adaptiveTechnologyTextField"
+              checkBoxName="adaptiveTechnology"
+              textFieldName="adaptiveTechnologyText"
+              checkBoxLabel={
+                t.reportAProblemPageDoesNotWorkWithAdaptiveTechnology
+              }
+              textFieldLabel={t.reportAProblemProvideMoreDetails}
+              uncontrolled={true}
+              multiText={true}
+              textLabelBold={true}
+              wrap="hard"
+              maxLength={750}
+            />
+            <OptionalTextField
+              checkBoxId="privacyIssues"
+              textFieldId="privacyIssuesTextField"
+              checkBoxName="privacyIssues"
+              textFieldName="privacyIssuesText"
+              checkBoxLabel={t.reportAProblemIncorrectInformation}
+              textFieldLabel={t.reportAProblemProvideMoreDetails}
+              uncontrolled={true}
+              multiText={true}
+              textLabelBold={true}
+              wrap="hard"
+              maxLength={750}
+            />
+            <OptionalTextField
+              checkBoxId="noWhereElseToGo"
+              textFieldId="noWhereElseToGoTextField"
+              checkBoxName="noWhereElseToGo"
+              textFieldName="noWhereElseToGoText"
+              checkBoxLabel={t.reportAProblemNoWhereElseToGo}
+              textFieldLabel={t.reportAProblemProvideMoreDetails}
+              uncontrolled={true}
+              multiText={true}
+              textLabelBold={true}
+              wrap="hard"
+              maxLength={750}
+            />
+            <OptionalTextField
+              checkBoxId="other"
+              textFieldId="otherTextField"
+              checkBoxName="other"
+              textFieldName="otherText"
+              checkBoxLabel={t.reportAProblemOther}
+              textFieldLabel={t.reportAProblemProvideMoreDetails}
+              uncontrolled={true}
+              multiText={true}
+              textLabelBold={true}
+              wrap="hard"
+              maxLength={750}
+            />
+          </fieldset>
+
+          <ul className="list-outside list-disc px-6 py-2">
+            <li className="text-xxs sm:text-sm font-body my-4 leading-tight sm:leading-6">
+              <b>{t.reportAProblemNoReply}</b>
+              {t.reportAProblemEnquiries}
+              <a
+                className="block text-xxs sm:text-sm font-body hover:underline hover:text-custom-blue-link focus:text-custom-blue-link focus:underline text-canada-footer-font"
+                href="mailto:experience@servicecanada.gc.ca"
+              >
+                experience@servicecanada.gc.ca
+              </a>
+            </li>
```

Comment:
I think this text is too small on mobile. There are a lot of different text sizes happening here 👇 . Let's see if it looks okay if it's the same size as the button.

<img width="300" alt="Screen Shot 2021-05-10 at 10 27 12" src="https://user-images.githubusercontent.com/2454380/117675230-61363100-b17a-11eb-8f7f-fd779848861c.png">